keyboards/ergodash/rev1/keymaps/ifohancroft/config.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+// place overrides here
+#undef USB_POLLING_INTERVAL_MS
```

Comment:
You shouldn't need to undef this, as it's not usually defined by default. 
```suggestion
```

Reply:
Ah! Thank you! Not sure why I thought it's defined by default.

Reply:
Done